modules/apps/object/object-api/src/main/java/com/liferay/object/petra/sql/dsl/DynamicObjectDefinitionTableUtil.java

```diff
@@ -46,8 +49,10 @@ public static String getAlterTableAddColumnSQL(
 		return sql;
 	}
 
-	public static String getDataType(String businessType, String dbType) {
-		String dataType = _dataTypes.get(dbType);
+	public static String getDataType(
+		String businessType, String dbColunmType, DBType dbType) {
```

Comment:
@pedro-oliveira446 do we have to pass it as a parameter? Couldn't we use the DBManagerUtil inside here?

Reply:
Just done!